docs/src/pages/components/sliderfield/demo.tsx

```diff
@@ -1,40 +1,69 @@
 import * as React from 'react';
+import { SliderField, SliderFieldProps } from '@aws-amplify/ui-react';
+import { Demo } from '@/components/Demo';
+import { SliderFieldPropControls } from './SliderFieldPropControls';
+import { useSliderFieldProps } from './useSliderFieldProps';
+import { getPropString } from '../utils/getPropString';
+import { demoState } from '@/utils/demoState';
 
-import { Flex, SliderField } from '@aws-amplify/ui-react';
+const propsToCode = (props: SliderFieldProps) => {
+  return (
+    `<SliderField` +
+    `\n  label="${props.label}"` +
+    (props.min ? `\n  min={${props.min}}` : '') +
+    `\n  max={${props.max}}` +
+    (props.step !== 1 ? `\n  step={${props.step}}` : '') +
+    getPropString(props.size, 'size') +
+    getPropString(props.trackSize, 'trackSize') +
+    getPropString(props.emptyTrackColor, 'emptyTrackColor') +
+    getPropString(props.filledTrackColor, 'filledTrackColor') +
+    getPropString(props.thumbColor, 'thumbColor') +
+    (props.orientation === 'vertical'
+      ? `\n  orientation="${props.orientation}"`
+      : '') +
+    (props.isDisabled ? `\n  isDisabled` : '') +
```

Comment:
previously we've tried to use the full `isDisabled={true}` for our examples instead of the shorthand `isDisabled` should we do the same here?